CAKE-setup.sh

```diff
@@ -0,0 +1,65 @@
+#! /bin/ash
+# Basic installation script for CAKE-autorate.sh
+# See https://github.com/lynxthecat/sqm-autorate for details
+# https://www.shellcheck.net/ is your friend
+
+#SRC_DIR="https://raw.githubusercontent.com/lynxthecat/CAKE-autorate/main/"
```

Comment:
```suggestion
SRC_DIR="https://raw.githubusercontent.com/lynxthecat/CAKE-autorate/main/"
```

Reply:
Looks good. Don't forget the README needs to point to the CAKE-autorate-setup.sh

Reply:
That is... new "setup script" name in **lynxthecat** repo